local-cli/server/runServer.js

```diff
@@ -55,21 +59,32 @@ async function runServer(args: Args, config: ConfigT) {
   const serverInstance = await Metro.runServer({
```

Comment:
```
Error ---------- local-cli/server/runServer.js:59:48
Cannot call `Metro.runServer` with object literal bound to the first parameter because property `watch` is missing in `RunServerOptions` [1] but exists in object literal [2].

                                                     v
  59|   const serverInstance = await Metro.runServer({
  60|     config: {
  61|       ...config,
    :
  77|     watch: !args.nonPersistent,
  78|   });
      --^

References:
  metro/packages/metro/src/index.js:201:4
  201| }: RunServerOptions) => {
          ^^^^^^^^^^^^^^^^ [1]
                                                     v
  59|   const serverInstance = await Metro.runServer({
  60|     config: {
  61|       ...config,
    :
  77|     watch: !args.nonPersistent,
  78|   });
      --^ [2]
```